montecarlo/mcpy/utils.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+def cross_product(X1, X2):
```

Comment:
How does this differ from the existing cross_product in our utilities?  Is this just so that montecarlo doesn't directly reference anything else in econml?

Reply:
Vestigial piece from metrics the previous framework calculating I think; we can remove it.